src/components/ScaffoldVuer.vue

```diff
@@ -1917,7 +1917,14 @@ export default {
                     annotationFeaturesToPrimitives(this.$module.scene, payload.features);
                   }
                 });
+                //Support previously supported encoded resource
+                getDrawnAnnotations(this.annotator, this.userToken, encodeURIComponent(this.url)).then((payload) => {
+                  if (payload && payload.features) {
+                    annotationFeaturesToPrimitives(this.$module.scene, payload.features);
+                  }
+                });
               }
+              this.activeDrawMode = "Edit";
```

Comment:
Need to place this inside nextTick. The style does not update correctly when switch to annotation mode.
```
this.$nextTick(() => {
  this.activeDrawMode = "Edit";
})
```